src/js/intlTelInput.js

```diff
@@ -1059,6 +1061,7 @@ class Iti {
 
   // close the dropdown and unbind any listeners
   _closeDropdown() {
+    this.selectedFlag.setAttribute('aria-expanded', false);
```

Comment:
Looks like this value is already set a couple of lines below, so we shouldn’t need this change. I suggest removing it.

Reply:
@agarzola thanks I just removed the offending line.